README.md

```diff
@@ -21,8 +21,9 @@ Read the blog post about this repository at:
 ## Installation
 
 ```
-$ composer require broadway/broadway ~0.2.0
+$ composer require broadway/broadway
 ```
+[packagist]: https://packagist.org/packages/broadway/broadway
```

Comment:
Sorry, I missed this when I reviewed it earlier, but you define a link which isn't used anywhere...